src/lib/v2/mappers/local/sidebarTree.tsx

```diff
@@ -86,13 +86,16 @@ function getWorkspaceChildrenOrdered(
   sortingOrder: SidebarTreeSortingOrder,
   workspaceRows: LocalTreeItem[]
 ): LocalTreeItem[] {
+  const compare = (a: LocalTreeItem, b: LocalTreeItem) => {
```

Comment:
I would call it a bit more descriptive, for example what are we comparing, by which algorithm does this comparison function sorts items etc.